akka-docs/src/test/java/jdocs/stream/StreamTestKitDocTest.java

```diff
@@ -102,7 +99,7 @@ public void pipeToTestProbe() throws Exception {
       .grouped(2)
       .runWith(Sink.head(), mat);
     akka.pattern.PatternsCS.pipe(future, system.dispatcher()).to(probe.ref());
-    probe.expectMsg(FiniteDuration.create(3, TimeUnit.SECONDS),
+    probe.expectMsg(scala.concurrent.duration.FiniteDuration.create(3, TimeUnit.SECONDS),
```

Comment:
I think we have an `expectMsg` that takes a Java duration nowadays, right?

Reply:
Done